addons/confirma/src/scenes/confirma_autoload/ConfirmaAutoload.cs

```diff
@@ -57,6 +57,15 @@ private bool CheckArguments()
                 _usedConfirmaApi = true;
             }
 
+            if (arg.StartsWith(prefix + "help", InvariantCulture))
+            {
+                _ = GetTree().CallDeferred("change_scene_to_file", "uid://tx5xyqboqit7");
```

Comment:
This is causing the issue, but why?
Same approach is also used in line 237 but that isn't causing any error.

Reply:
Changing `return true;` to `return false;` seems to fix the issue.

Reply:
ok, this should be fixed in bb9ffbeb